File: easy/small_factorial/small_factorial.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Sunday, 17. November 2013 11:36PM
from math import factorial

def main():
    iterations = int(raw_input())
    for inp in range(iterations):
        print factorial(int(raw_input()))

if __name__ == '__main__':
    main()

File: easy/small_factorial/small_factorial.c
/*
    started at: Sunday, 17. November 2013 11:36PM
*/

// NOT WORKING YET!

#include <stdio.h>

int factorial(int number)
{
    int fact = 1;
    while (number)
    {
        fact *= number;
        number--;
    }

    return fact;
}

int main(int argc, char const *argv[])
{
    int iterations;
    int number;

    scanf("%d",&iterations);

    for (int i = 0; i < iterations; ++i)
    {   
        scanf("%d",&number);
        printf("%d\n",factorial(number)); 
    }
    return 0;
}
